lib/ace/ext/beautify_test.js

```diff
@@ -337,7 +338,21 @@ module.exports = {
             + "\t\tpadding: 10px;\n"
             + "\t}\n"
             + "</style>");
-
+        
+        beautify.beautify(s);
```

Comment:
Why this and not `next()`?

Reply:
next is not needed for synchronous tests, i probably should have removed all uses of next in this file.

Reply:
Ok. Although `beautify.beautify(s)` call seems redundant here, since you've already done this and checked the results several lines above.

Reply:
ah you are right, i misunderstood your question, will fix in a followup pr.